addons/website/static/src/snippets/s_share/000.js

```diff
@@ -4,18 +4,24 @@ import publicWidget from 'web.public.widget';
 
 const ShareWidget = publicWidget.Widget.extend({
     selector: '.s_share, .oe_share', // oe_share for compatibility
-    events: {
+    read_events: {
         'click a': '_onShareLinkClick',
     },
+    disabledInEditableMode: false,
```

Comment:
Most of the time, I don't find this ok to change this in stable. If someone made a custo, their custo will suddenly be enabled in edit mode as well.